src/Symfony/Component/Intl/ResourceBundle/LanguageBundle.php

```diff
@@ -42,15 +42,6 @@ public function __construct(string $path, BundleEntryReaderInterface $reader, Lo
      */
     public function getLanguageName($language, $region = null, $displayLocale = null)
     {
-        // Some languages are translated together with their region,
-        // i.e. "en_GB" is translated as "British English"
-        if (null !== $region) {
-            try {
-                return $this->getName($language.'_'.$region, $displayLocale);
-            } catch (MissingResourceException $e) {
-            }
```

Comment:
from the issue

> // should return `Austrian German` does return `German`

so the fallback already happens, due this catch. This PR doesnt make sense :sweat_smile: